src/main/java/zd/zero/waifu/motivator/plugin/promotion/AniMemePluginPromotionRunner.kt

```diff
@@ -60,18 +61,20 @@ object AniMemePluginPromotion {
                         .toOptional()
                         .filter { it.isNotEmpty() }
                         .map { it.first() }
-                        .map {
-                            WindowManager.getInstance().suggestParentWindow(
-                                it
+                        .map { project: Project ->
+                            Pair(
+                                WindowManager.getInstance().suggestParentWindow(project),
```

Comment:
I think this was part of the optional chain because `WindowManager.getInstance().suggestParentWindow(project)` returns a nullable response.

Previously if it was null, it would not execute the promotion closure, but the rejection function.

Let's make sure that a non-null window is passed to the promotion dialog.

Reply:
done